### PR TITLE
Fix samples that rely on deprecated namespaces

### PR DESCRIPTION
**Pull request by networkfusion**
## Description
* Samples will fail to work with the deprecated `Windows.Devices` namespace.

## Motivation and Context
- Fixes nanoFramework/Home#620

## How Has This Been Tested?
Requires further testing... This PR just works towards a sucessful compile.

## Types of changes

- [ ] Improvement (non-breaking change that improves a sample)
- [ ] Bug fix (fixes an issue with a current sample)
- [ ] New Sample (adds a new sample)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Signed-off-by: GITHUB_USER <GITHUB_USER_EMAIL>

### CODE REVIEW

**Review by Ellerbach — APPROVED** [review]
Thanks a lot for this work! Great catch on many of the samples and finding hidden ones!

**Comment by josesimoes** [comment]
@networkfusion any pending work on this PR or it can be merged?

**Comment by networkfusion** [comment]
> @networkfusion any pending work on this PR or it can be merged?

@josesimoes I plan no further work, although I was hoping that a member of the community would test each of the ones changed to make sure I havent missed something (very likely) and they work. However, I guess this can be merged and other issues raised if something has been forgotten...

File: samples/HTTP/HttpAzureGET/Program.cs [hunk 1]
```diff
@@ -59,8 +59,7 @@ public static void Main()
             }
 
             // setup user button
-            _userButton = GpioController.GetDefault().OpenPin(0);
-            _userButton.SetDriveMode(GpioPinDriveMode.Input);
+            _userButton = GpioController.OpenPin(0, PinMode.Input);
```
Comment (networkfusion) [inline samples/HTTP/HttpAzureGET/Program.cs:62]: _userButton = new GpioController().OpenPin(0, PinMode.Input);

**Comment by networkfusion** [comment]
Only other (probable) issue is the sample [samples/Hardware.Stm32/Stm32.PowerMode/Program.cs](https://github.com/nanoframework/Samples/pull/209/files#diff-edb021d6b4803d78c165b7fe03e709f0163604f37d15cd36f238453d2992dc00)

**Comment by networkfusion** [comment]
Extra notes: The pipeline does not report build failures properly: e.g. 
image

There are many projects that still rely on `Windows.Storage` and `Windows.Storage.Streams` (which were deprecated and removed).

**Comment by networkfusion** [comment]
It turns out that Azure Pipelines is pretty poor when it comes to reporting actual errors with solution builds... Although I can continue working on the solutions, I am hoping I have now "caught" all of the deprecated nugets...

**Comment by josesimoes** [comment]
All samples are now building correctly. Exception goes to FileAccess which need a serious rework. I've disabled the project in solution waiting that this is reworked.

**Pull request merged**